runtime/runtime/src/tests/apply.rs

```diff
@@ -2793,3 +2837,169 @@ fn test_deploy_and_call_local_receipts() {
         ActionErrorKind::FunctionCallError(FunctionCallError::MethodResolveError(_))
     );
 }
+
+#[test]
+fn test_transaction_ordering_with_apply() {
+    let alice_key1 = InMemorySigner::test_signer(&alice_account());
+    let bob_signer = InMemorySigner::test_signer(&bob_account());
+    let alice_key2 = InMemorySigner::from_seed(alice_account(), KeyType::ED25519, "seed");
+
+    let tx1 = SignedTransaction::send_money(
+        1,
+        alice_account(),
+        alice_account(),
+        &alice_key2, // invalid access key, tx should be dropped
```

Comment:
is `alice_key2` invalid?  Why not rename the variable to `alice_invalid_key` then?

Reply:
done